src/content/reference/manifest.mdx

```diff
@@ -752,6 +763,20 @@ The name of the endpoint.
 
 The url of the endpoint.
 
+> Dynamic endpoints: The url can be set during deployment stage by sharing `$OKTETO_EXTERNAL_{EXTERNAL_NAME}_ENDPOINTS_{ENDPOINT_NAME}_URL`
+
+#### icon (string, optional)
+
+ Sets the icon that will be shown in the UI. The supported values for icons are:
```

Comment:
```suggestion
 Sets the icon that will be shown in the UI. The supported values for icons are listed below. If empty, it will default to `default`.
```